lib/config.js

```diff
@@ -141,6 +142,9 @@ Config.prototype.buildArgs = function () {
     args.brave_ads_enabled = false
     args.enable_tor = false
     args.enable_brave_sync = false
+    if (!this.officialBuild) {
```

Comment:
maybe place this above TODO(fixme) so it doesn't get confused with the stuff we have temporarily disabled?